octavia-cli/octavia_cli/list/connectors_definitions.py

```diff
@@ -0,0 +1,121 @@
+            data (List[List[str]]): Tabular data containing rows and columns.
+            padding (int): Number of character to adds to create space between columns.
+        Returns:
+            col_width (int): The computed column width according to input data.
+        """
+        col_width = max(len(col) for row in data for col in row) + padding
+        return col_width
+
+    # TODO alafanechere: declare in a specific formatting module because it will probably be reused
+    @staticmethod
+    def _camelcased_to_uppercased_spaced(camelcased: str) -> str:
+        """Util function to transform a camelCase string to a UPPERCASED SPACED string
+        e.g: dockerImageName -> DOCKER IMAGE NAME
+        Args:
+            camelcased (str): The camel cased string to convert.
+
+        Returns:
+            (str): The converted UPPERCASED SPACED string
+        """
+        return "".join(map(lambda x: x if x.islower() else " " + x, camelcased)).upper()
+
+    # TODO alafanechere: declare in a specific formatting module because it will probably be reused
+    @staticmethod
+    def _display_as_table(data: List[List[str]]) -> str:
+        """Formats tabular input data into a displayable table with columns.
+        Args:
+            data (List[List[str]]): Tabular data containing rows and columns.
+        Returns:
+            table (str): String representation of input tabular data.
+        """
+        col_width = ConnectorsDefinitions._compute_col_width(data)
+        table = "\n".join(["".join(col.ljust(col_width) for col in row) for row in data])
+        return table
+
+    # TODO alafanechere: declare in a specific formatting module because it will probably be reused
+    @staticmethod
+    def _format_column_names(camelcased_column_names: List[str]) -> List[str]:
+        """Format camel cased column names to uppercased spaced column names
+
+        Args:
+            camelcased_column_names (List[str]): Column names in camel case.
+
+        Returns:
+            (List[str]): Column names in uppercase with spaces.
+        """
+        return [ConnectorsDefinitions._camelcased_to_uppercased_spaced(column_name) for column_name in camelcased_column_names]
+
+    def __repr__(self):
+        definitions = [self._format_column_names(self.fields_to_display)] + self.latest_definitions
+        return self._display_as_table(definitions)
+
+
+class SourceConnectorsDefinitions(ConnectorsDefinitions):
+    api = source_definition_api.SourceDefinitionApi
+
+    def __init__(self, api_client: airbyte_api_client.ApiClient):
+        super().__init__(DefinitionType.SOURCE, api_client, self.api.list_latest_source_definitions)
+
+
+class DestinationConnectorsDefinitions(ConnectorsDefinitions):
+    api = destination_definition_api.DestinationDefinitionApi
+
+    def __init__(self, api_client: airbyte_api_client.ApiClient):
+        super().__init__(DefinitionType.DESTINATION, api_client, self.api.list_latest_destination_definitions)
```

Comment:
I like how you did this inheritance setup - you were able to avoid repeating 99% of the shared logic between source and destination definitions. Nice job!